managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryImpl.java

```diff
@@ -47,12 +48,22 @@ protected EntryImpl newObject(Handle<EntryImpl> handle) {
     ByteBuf data;
 
     public static EntryImpl create(LedgerEntry ledgerEntry) {
+        return create(ledgerEntry,null);
+    }
+
+    public static EntryImpl create(LedgerEntry ledgerEntry, ManagedLedgerInterceptor managedLedgerInterceptor) {
```

Comment:
Put the 'managedLedgerInterceptor' in here seem a little strange. 
IMHO, it's better put outside of `create`, the `managedLedgerInterceptor` processes the `ledgerEntry` and then pass the result to this create method.

Reply:
I extended the create() to ensure that future usages of this method from other places don't miss out the interceptor. 
I do agree that it will be clean to have the calling methods process the ledger entry using interceptors and pass the result.